charts/fleet-agent/templates/patch_default_serviceaccount.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: patch-fleet-sa
+  annotations:
+    "helm.sh/hook": post-install, post-upgrade
+    "helm.sh/hook-delete-policy": hook-succeeded, before-hook-creation
+spec:
+  template:
+    spec:
+      serviceAccountName: fleet-agent
+      restartPolicy: Never
+      containers:
+      - name: sa
+        image: "{{ template "system_default_registry" . }}{{ .Values.global.kubectl.repository }}:{{ .Values.global.kubectl.tag }}"
```

Comment:
Why patching it instead of adding it directly to the service account?

Reply:
oh nvm

Reply:
this is the default serviceaccount auto-created by kubernetes after helm creates the namespace